compiler-rt/lib/ctx_profile/tests/CtxInstrProfilingTest.cpp

```diff
@@ -20,3 +29,173 @@ TEST(ArenaTest, Basic) {
   Arena::freeArenaList(A);
   EXPECT_EQ(A, nullptr);
 }
+
+TEST_F(ContextTest, Basic) {
+  auto *Ctx = __llvm_ctx_profile_start_context(&Root, 1, 10, 4);
+  EXPECT_NE(Ctx, nullptr);
+  EXPECT_NE(Root.CurrentMem, nullptr);
+  EXPECT_EQ(Root.FirstMemBlock, Root.CurrentMem);
+  EXPECT_EQ(Ctx->size(), sizeof(ContextNode) + 10 * sizeof(uint64_t) +
```

Comment:
`Ctx->size()` will segfault even if the expectation on L35 fails since that does not fail the test. L35 should be an ASSERT.

Reply:
done.